pipelines/operator-hosted-pipeline.yml

```diff
@@ -18,9 +20,15 @@ spec:
       default: quay.io
     - name: image_stream
       default: $(context.pipelineRun.namespace)
+    - name: test_mode
```

Comment:
The test mode should be only in the ci pipeline to allow quicker bundle development, hosted should always run all tasks.

Reply:
I set the "skip" to default "false" in the `diggest-pinning` task and removed this argument from the hosted pipeline.